.github/workflows/doc-tests.yaml

```diff
@@ -142,6 +142,26 @@ jobs:
         working-directory: 'docs'
         run: yarn markdown-lint
 
+      - name: Check out vale
```

Comment:
Do you actually need the vale checkout for anything? Or is it just for the `gh release download` command? If the latter, you can run `gh release download -R errata-ai/vale ...`